hpy/devel/src/runtime/ctx_tuplebuilder.c

```diff
@@ -0,0 +1,53 @@
+#include <stddef.h>
+#include <Python.h>
+#include "hpy.h"
+
+#ifdef HPY_UNIVERSAL_ABI
+   // for _h2py and _py2h
+#  include "handles.h"
+#endif
+
+
+_HPy_HIDDEN HPyTupleBuilder
+ctx_TupleBuilder_New(HPyContext ctx, HPy_ssize_t initial_size)
```

Comment:
should it be just `size`? It's not that it can change anyway

Reply:
Right now yes, but the C API gives `_PyTuple_Resize()`.  You can resize tuples while you build them (and you're not supposed to use that after the tuple is exposed to Python, of course).

Reply:
ah, I didn't know. The question is whether we want to expose this functionality in `TupleBuilder`. But I am fine for not having to discuss it now and leave the argument name as it is